decode_data.py

```diff
@@ -24,14 +24,12 @@ def decode1(data: List[int]) -> Dict[int, Any]:
         5: data[7]
     }
 
-def decode2(data: List[int]) -> Dict[int, Any]:
+def decodeBMSStatus(data: List[int]) -> Dict[int, Any]:
     return {
-        6: data[0],
-        7: data[1],
-        8: pd.bigEndian(data[2:4]),
-        9: pd.bigEndian(data[4:6]),
-        10: pd.twosComp(data[6], 8),
-        11: pd.twosComp(data[7], 8)
+        106: data[0],
+        107: pd.bigEndian(data[1:5]),
```

Comment:
Is there a reason for this being big endian? I don't think it will be broadcast that way, but @dyldonahue could confirm